ibm_mq/assets/configuration/spec.yaml

```diff
@@ -166,6 +166,12 @@ files:
         value:
           example: False
           type: boolean
+      - name: try_normal_ssl
+        description: Whether or not to try a normal connection before SSL when `ssl_auth` is enabled.
```

Comment:
```suggestion
        description: Enable this option to try a normal connection before SSL when `ssl_auth` is enabled.
```

Reply:
Since it's disabled by default I don't think we should use this wording